tox.ini

```diff
@@ -23,8 +22,9 @@ commands =
     pre-commit install -f --install-hooks
 
 [testenv:pre-commit]
+basepython = python3.7
 deps = pre-commit >= 1.16.1
-commands = pre-commit {posargs}
+commands = pre-commit run --all-files
```

Comment:
Why shuffle this? I think it's pretty reasonable to have pre-commit in `make test`.

Reply:
Oh I see, it's now its own environment, ignore me :)